confmap/README.md

```diff
@@ -99,16 +118,62 @@ configuration retrieved via the `Provider` used to retrieve the “initial” co
             │      onChange       │
             │◄────────────────────┤
 ◄───────────┤                     │
+            |                     |
+  Resolve   │                     │
+───────────►│                     │
+            │                     │
+            │      Retrieve       │
+            ├────────────────────►│
+            │        Conf         │
+            │◄────────────────────┤
+◄───────────┤                     │
+```
+
+Example of `Provider` with `onChange` func that periodically gets notified:
```

Comment:
I wonder if we could turn this into [a testable example](https://go.dev/blog/examples). That way we are able to lint and test this code over time and ensure it doesn't get outdated (which is harder in a Markdown file)

Reply:
Thank you!